---
title: "' Throw ' işleneni ' System. Exception ' öğesinden türetilmelidir"
ms.date: 07/20/2015
f1_keywords:
- vbc30665
- bc30665
helpviewer_keywords:
- BC30665
ms.assetid: 7c228087-39ea-4b30-a410-6ba711e67e5e
ms.openlocfilehash: 92fccbd2faac5580d28fef8cc02b51b23715f136
ms.sourcegitcommit: f8c270376ed905f6a8896ce0fe25b4f4b38ff498
ms.translationtype: MT
ms.contentlocale: tr-TR
ms.lasthandoff: 06/04/2020
ms.locfileid: "84414952"
---
# <a name="throw-operand-must-derive-from-systemexception"></a>' Throw ' işleneni ' System. Exception ' öğesinden türetilmelidir
İçin sağlanan bağımsız değişken, ' `Throw` ın bir örneği `System.Exception` ya da öğesinden türetilmiş bir sınıfın bir örneği olmalıdır `System.Exception` .  
  
 **Hata kimliği:** BC30665  
  
## <a name="to-correct-this-error"></a>Bu hatayı düzeltmek için  
  
- Aşağıdaki örnekte olduğu gibi, öğesinden türetilmiş bir bağımsız değişken kullanın `System.Exception` .  
  
    ```vb
    Throw New System.Exception("This is an error.")  
    ```  
  
## <a name="see-also"></a>Ayrıca bkz.

- [Throw Deyimi](../language-reference/statements/throw-statement.md)
- [Try...Catch...Finally Deyimi](../language-reference/statements/try-catch-finally-statement.md)
- <xref:System.Exception?displayProperty=nameWithType>
- [.NET 'te özel durumları işleme ve atma](../../standard/exceptions/index.md)
